src/mlpack/methods/perceptron/perceptron_main.cpp

```diff
@@ -47,10 +47,17 @@ PROGRAM_INFO("Perceptron",
     " parameter), or both those things at once.  In addition, this program "
     "allows classification on a test dataset (via the " +
     PRINT_PARAM_STRING("test") + " parameter) and the classification results "
-    "on the test set may be saved with the " + PRINT_PARAM_STRING("output") +
-    "output parameter.  The perceptron model may be saved with the " +
+    "on the test set may be saved with the " +
+    PRINT_PARAM_STRING("predictions") +
+    " output parameter.  The perceptron model may be saved with the " +
     PRINT_PARAM_STRING("output_model") + " output parameter."
     "\n\n"
+    "Note : The following parameter is deprecated and "
+    "will be removed in mlpack 4: " + PRINT_PARAM_STRING("output") +
+    "."  +
+    "\nUse " + PRINT_PARAM_STRING("predictions") + " instead of " +
+    PRINT_PARAM_STRING("output")+'.'
```

Comment:
Missing `+` at the end. Also missing spaces around the `+`.

Reply:
Rectified, Also made same changes in adaboost file too, didn't want to split pr for small changes.